livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -359,25 +359,7 @@ private static void escapeJavaStyleString(Writer out, String str) throws IOExcep
             break;
         }
       } else {
-        switch (ch) {
-          case '\'' :
-            out.write('\\');
-            break;
-          case '"' :
-            out.write('\\');
-            out.write('"');
-            break;
-          case '\\' :
-            out.write('\\');
-            out.write('\\');
-            break;
-          case '/' :
-            out.write('\\');
-            break;
```

Comment:
I believe at least that escaping for `'` and `/` is not proper, but I don't know what's the original intention of the author

Reply:
At present, this transformation is meaningless and instead brings many difficulties to data processing.

Reply:
This is due to this ticket. https://issues.apache.org/jira/browse/ZEPPELIN-5458
@myongyun Could you help check whether Chinese character can be displayed correctly after your PR? Thanks

Reply:
@myongyun Could you please verify the @zjffdu's comments?